<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EnergyMinimizationDock</class>
 <widget class="QDockWidget" name="EnergyMinimizationDock">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>213</width>
    <height>270</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Energy Minimization</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="forceFieldLabel">
      <property name="text">
       <string>Force Field:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QComboBox" name="forceFieldSelector">
      <item>
       <property name="text">
        <string>UFF</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>AMBER</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>MMFF</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>OPLS</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="1" column="0" colspan="2">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="energyLabel">
      <property name="text">
       <string>Energy:</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLabel" name="energyText">
      <property name="text">
       <string>Unknown</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="statusLabel">
      <property name="text">
       <string>Status:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLabel" name="statusText">
      <property name="text">
       <string>Stopped</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0" colspan="2">
     <widget class="QPushButton" name="startButton">
      <property name="text">
       <string>Start</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
